deployment/terraform/cloudwatch.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_cloudwatch_log_group" "app" {
+  #checkov:skip=CKV_AWS_158:TODO-Add encryption to cloudwatch log group
```

Comment:
This will need to be a separate PR to try to implement [CKV_AWS_158](https://docs.bridgecrew.io/docs/ensure-that-cloudwatch-log-group-is-encrypted-by-kms), and we need to figure out if we should make a separate key for it for the account or use a separate key per application.